man/subscription-manager.8

```diff
@@ -384,7 +384,7 @@ for the system.
 
 .TP
 .B --show
-Shows the system's current set of syspurpose preference. This is output in the form of a blob of json. Single-valued entries for which there is no value will be included in the output with a value of "". List entries which have no value will be included in the output with a value of "[]" (less the quotes).
+Shows the system's current set of syspurpose preference, formatted as JSON. Single-valued entries for which there is no value will be included in the output with a value of "". List entries which have no value will be included in the output with a value of "[]" (less the quotes).
```

Comment:
English is not my first language, but shouldn't be the change just "formatted as JSON" (without comma at beginning)?

Reply:
Me neither :) I added the comma to ease a bit the reading of the sentence, otherwise it'd be a single non-trivial sentence without any pause (comma, semicolon, colon, etc).

Reply:
@tlhmerry9 I would like to know opinion, because you are native speaker. :smiley:

Reply:
I think it reads better:
Shows the system's current set of syspurpose preferences formatted as JSON.